cmd/vcsstore/vcsstore.go

```diff
@@ -155,15 +153,15 @@ The options are:
 	vh.Log = log.New(logw, "server: ", log.LstdFlags)
 	vh.Debug = *debug
 
-	if *datadNode {
```

Comment:
you can entirely remove datad in another PR if you want. commenting out is fine too.